config.toml

```diff
@@ -116,4 +128,10 @@ themesDir = "node_modules/"
     name = "行動規範"
     url = "/japan2020/code-of-conduct/"
     weight = 1
+    parent = "行動指針"
+
+[[languages.jp.menu.japan2020]]
+    name = "Speaker Agreement"
+    url = "/speaker-agreement/"
```

Comment:
I think the url here should be /jp/speaker-agreement/

Reply:
If you check the menu on this page: https://deploy-preview-66--jakartaee-jakartaone.netlify.com/jp/japan2020/ the menu is smart enough to know its the JP page its linking to :)

Reply:
Awesome! Even better!